src/auth/user.controller.ts

```diff
@@ -1,12 +1,37 @@
-import { Controller, Get, Req, UseGuards } from '@nestjs/common';
+import {
+  Controller,
+  Get,
+  Req,
+  UseGuards,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
 import { AuthGuard } from '@nestjs/passport';
 import { ApiBearerAuth, ApiOperation, ApiTags } from '@nestjs/swagger';
 
+@Injectable()
+export class CustomJwtAuthGuard extends AuthGuard('jwt') {
+  handleRequest(err, user, info) {
+    if (err || !user) {
+      if (info?.name === 'TokenExpiredError') {
```

Comment:
info의 name보다는 이거는 어떤가요?
```suggestion
      if (err instanceof TokenExpiredError) {
```

Reply:
에러 이름이 아니라 인스턴스인지 확인하는 방식이네요! 새로 배워갑니다